File: online-judge-solutions/Codeforces/1719A.cpp
//https://codeforces.com/problemset/problem/1719/A
//Lang		:	GNU C++14
//Time		:	62 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
int main(){
	int t,n,m ;
	cin>>t;
	while(t--){
		cin>>n>>m;
		cout<<((n+m)%2?"Burenka":"Tonya")<<endl;
	}
	return 0;
}

File: online-judge-solutions/Codeforces/1719A.py
#https://codeforces.com/problemset/problem/1719/A
#Lang		:	Python 3.8
#Time		:	108 ms
#Memory         :	0 KB
for _ in range(int(input())):
    n,m=map(int,input().split())
    print("Burenka" if (n+m)%2 else "Tonya")
